api/zeta-environment.interfaces.ts

```diff
@@ -88,6 +88,14 @@ export interface ZetaProjectOptions {
      * holds options about how the user is able to login to the GUI - default: only credentials login and no language change
      */
     auth?: AuthenticationOptions;
+    /**
+     * extend the Privacy Link with the language parameter
+     */
+    getPrivacyLink?(lang: string): string;
+    /**
+     * if the link is set, then the privacy button will be displayed inside the login mask
+     */
+    privacyLink?: string;

```

Comment:
now you won't need this one, anymore, as it would be redundant to the `getPrivacyLink` function. Only use the above one and merge the respective documentation.

Reply:
to delete this was my first try, but i run into a problem with this function showPrivacyButton = !!environment.zeta.privacyLink; Iexplain more in a mail